mongodbatlas/resource_mongodbatlas_project_api_key.go

```diff
@@ -25,7 +25,7 @@ func resourceMongoDBAtlasProjectAPIKey() *schema.Resource {
 		Schema: map[string]*schema.Schema{
```

Comment:
@AgustinBettati as there are significant changes in this file, I don't know if it's ok for you to wait until I merge the file structure PR.

Reply:
As long as this is merged before coming release no hurry with this PR.